FlipperZero/Shared/Assets.xcassets/AccentColor.colorset/Contents.json

```diff
@@ -1,6 +1,33 @@
 {
   "colors" : [
     {
+      "color" : {
+        "color-space" : "display-p3",
+        "components" : {
+          "alpha" : "1.000",
+          "blue" : "0.263",
+          "green" : "0.576",
+          "red" : "0.935"
+        }
+      },
+      "idiom" : "universal"
+    },
+    {
+      "appearances" : [
+        {
+          "appearance" : "luminosity",
+          "value" : "dark"
+        }
+      ],
+      "color" : {
+        "color-space" : "display-p3",
+        "components" : {
+          "alpha" : "1.000",
+          "blue" : "0.118",
+          "green" : "0.118",
+          "red" : "0.118"
```

Comment:
This seems to be too dark, it's hard to recognize on the dark background
![Screen Shot 2020-10-01 at 06 19 20](https://user-images.githubusercontent.com/10916937/94814336-0cddfe00-03ae-11eb-9756-a87b72fba10f.png)